README.md

```diff
@@ -1,2 +1,46 @@
 # rest-api-express
-Management web app
+
+**Description**  
+CRUD app for products (en - es)
+
+**Technologies**
++ NodeJS Express (including dependencies)
++ MySQL
++ JavaScript
++ Html
++ CSS
+
+# start server
+
+1. **Execute in terminal to install node-modules**
+```
+npm i --yes
+```  
+  
+2. **You will need a database in MySQL (XAMMP) and connect it. Use a .env file to enter the database data. INSTRUCTIONS BELOW**. 
+
+3. **Run server in terminal**
+```
+npm run dev
+```
+
+4. Finally, run app in: http://localhost:3000/
+
+
+# database instructions
+
+**Installing and configuration**
+
+1. Create the host, user, password, and database in XAMPP.
+2. Enter the data in a .env file **(you will find a .env.example)**. 
+  
+**Database estructure**  
```

Comment:
it would more helpful if you paste the commands needed to create the database and table
```
CREATE DATABASE expressdb;

CREATE TABLE new_product (
     product_id MEDIUMINT NOT NULL AUTO_INCREMENT,
     product_name CHAR(30) NOT NULL,
     product_price DOUBLE NOT NULL,
     product_detail CHAR(30) NOT NULL,
     product_img CHAR(200) NOT NULL,
     PRIMARY KEY (product_id)
);
```